### PR TITLE
Added strong ciphers by default in kube-apiserver and kubelet

### PR DESCRIPTION
**Pull request by sachinkumarsingh092**
#### Summary

Added `--tls-cipher-suite` arg in kube-apiserver and kubelet. This is done to have this option by default, which is usually applied when CIS hardening your cluster.

#### Testing

Build the snap locally and verified the presence of the argument in the respective args file.

### CODE REVIEW

File: microk8s-resources/default-args/kubelet [hunk 1]
```diff
@@ -11,3 +11,4 @@
 --node-labels="microk8s.io/cluster=true,node.kubernetes.io/microk8s-controlplane=microk8s-controlplane"
 --authentication-token-webhook=true
 --read-only-port=0
+--tls-cipher-suites=TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_128_GCM_SHA256
```
Comment (neoaggelos) [inline microk8s-resources/default-args/kubelet:14]: please include newline at end of file

**Review by neoaggelos — COMMENTED** [review]
Looks OK, left a minor comment. How is this tested?

File: microk8s-resources/default-args/kubelet (hunk 1 above)
Comment (sachinkumarsingh092) [reply microk8s-resources/default-args/kubelet:14]: Done.

**Comment by sachinkumarsingh092** [comment]
For testing, I build and checked the presence of the args inside the respective args files.

**Comment by ktsakalozos** [comment]
LGTM +1

**Pull request merged**